src/editors/data/redux/thunkActions/requests.test.js

```diff
@@ -101,77 +113,64 @@ describe('requests thunkActions module', () => {
       });
     });
   };
-
   describe('network request actions', () => {
-    const submissionUUID = 'test-submission-id';
-    const locationId = 'test-location-id';
+    const sometestStateGetter = () => ({
```

Comment:
this should really just be:
```
const getState = () => testState;
```
then, each of your tests against selectors just check against the object, not by calling the getter again.